### PR TITLE
Added file size string response

### PR DESCRIPTION
**Pull request by MobCat**
Added a ParseResponse for the formatted file size string.

### CODE REVIEW

File: src/anonfile/anonfile.py [hunk 1]
```diff
@@ -146,9 +146,18 @@ def name(self) -> Path:
     @property
     def size(self) -> int:
         """
-        Return the uploaded file size in bytes.
+        Return the uploaded file size in bytes as an integer.
         """
         return int(self.json['data']['file']['metadata']['size']['bytes'])
+    
+    # It was the only dedicated response missing from the original json query
```
Comment (hentai-chan) [inline src/anonfile/anonfile.py:153]: You should remove this comment, put it in a commit message if you want to but don't add these remarks in the code directly.

File: src/anonfile/anonfile.py [hunk 2]
```diff
@@ -146,9 +146,18 @@ def name(self) -> Path:
...
+    # So I just added it. - MobCat
+    @property
+    def sizeStr(self) -> str:
```
Comment (hentai-chan) [inline src/anonfile/anonfile.py:156]: Method names in Python should always be written in snake_case.

**Review by hentai-chan — CHANGES_REQUESTED** [review]
Small changes are required, but other than that the code looks good to me.

**Comment by MobCat** [comment]
Sorry for the late reply. I've never done a code push on github before. So if possible please make any changes you need to. I don't really know what I have to do on my end.

**Comment by hentai-chan** [comment]
You just need to make a new commit on the repository where this pull request stems from.

**Comment by hentai-chan** [comment]
@MobCat Hey ya, just chiming in to check if you are still working on this PR? If not I can add the due changes myself in the coming days, so your work isn't lost. I will leave this up during the weekend and come back on Monday to see if this branch is still stale, else I will consider it up for grabs :)

**Comment by StefanGreve** [comment]
Closed by #80

**Pull request closed without merging**